From 61bf9c0b727f17181017ac1fafaf1130dbb2a182 Mon Sep 17 00:00:00 2001
From: Aaron Armstrong Skomra <skomra@gmail.com>
Date: Wed, 31 May 2017 09:02:19 -0700
Subject: [PATCH libwacom 4/7] data: Add .tablet files for Wireless Intuos Pro
 second generation

Also correct width and height for intuos-pro-2-l.tablet,
cintiq-pro-13.tablet and cintiq-pro-16.tablet.

Signed-off-by: Aaron Armstrong Skomra <aaron.skomra@wacom.com>
Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
(cherry picked from commit ecf0bf4edde53f8f08c01cdc36f4e6a2fa7ffa65)
---
 data/cintiq-pro-13.tablet     |  4 +--
 data/cintiq-pro-16.tablet     |  4 +--
 data/intuos-pro-2-l-wl.tablet | 67 +++++++++++++++++++++++++++++++++++++++++++
 data/intuos-pro-2-l.tablet    |  4 +--
 data/intuos-pro-2-m-wl.tablet | 67 +++++++++++++++++++++++++++++++++++++++++++
 5 files changed, 140 insertions(+), 6 deletions(-)
 create mode 100644 data/intuos-pro-2-l-wl.tablet
 create mode 100644 data/intuos-pro-2-m-wl.tablet

diff --git a/data/cintiq-pro-13.tablet b/data/cintiq-pro-13.tablet
index 197ad3f..e9fb025 100644
--- a/data/cintiq-pro-13.tablet
+++ b/data/cintiq-pro-13.tablet
@@ -31,8 +31,8 @@ Name=Wacom Cintiq Pro 13
 Class=Cintiq
 DeviceMatch=usb:056a:034F
 PairedID=usb:056a:0353
-Width=14
-Height=9
+Width=12
+Height=7
 # No pad buttons, so no layout
 Styli=0x842;0x84a;0x160802;0x16080a;0x100802;0x10080a;0x140802;0x14080a;0x120802;0x100804;0x10080c;0x100902;0x10090a
 IntegratedIn=Display
diff --git a/data/cintiq-pro-16.tablet b/data/cintiq-pro-16.tablet
index e64bdd3..6a8e3fa 100644
--- a/data/cintiq-pro-16.tablet
+++ b/data/cintiq-pro-16.tablet
@@ -31,8 +31,8 @@ Name=Wacom Cintiq Pro 16
 Class=Cintiq
 DeviceMatch=usb:056a:0350
 PairedID=usb:056a:0354
-Width=16
-Height=10
+Width=14
+Height=8
 # No pad buttons, so no layout
 Styli=0x842;0x84a;0x160802;0x16080a;0x100802;0x10080a;0x140802;0x14080a;0x120802;0x100804;0x10080c;0x100902;0x10090a
 IntegratedIn=Display
diff --git a/data/intuos-pro-2-l-wl.tablet b/data/intuos-pro-2-l-wl.tablet
new file mode 100644
index 0000000..89445e1
--- /dev/null
+++ b/data/intuos-pro-2-l-wl.tablet
@@ -0,0 +1,67 @@
+# Wacom
+# Intuos Pro 2 L WL
+# PTH-860
+#
+# Button Map:
+# (A=1, B=2, C=3, ...)
+#
+#    *-----------------------*
+#    |                       |
+#  A |                       |
+#  B |                       |
+#  C |                       |
+#  D |                       |
+#  I |        TABLET         |
+#  E |                       |
+#  F |                       |
+#  G |                       |
+#  H |                       |
+#    |                       |
+#    *-----------------------*
+#
+# Touch Ring Map:
+# (A=1st ring, B=2nd ring, ...)
+#
+#    *-----------------------*
+#    |                       |
+#  A |        TABLET         |
+#    |                       |
+#    *-----------------------*
+#
+# LED Map:
+# (XY=Bank X, LED Y; *=Invisible)
+#
+#        *-----------------------*
+#        |                       |
+#        |                       |
+#  01 02 |                       |
+#        |        TABLET         |
+#  00 03 |                       |
+#        |                       |
+#        |                       |
+#        *-----------------------*
+#
+
+[Device]
+Name=Wacom Intuos Pro 2 L WL
+DeviceMatch=bluetooth:056a:0361
+Class=Intuos5
+Width=12
+Height=8
+Layout=intuos-pro-2-l.svg
+IntegratedIn=
+Styli=0x842;0x84a;0x160802;0x16080a;0x100802;0x10080a;0x140802;0x14080a;0x120802;0x100804;0x10080c;0x100902;0x10090a
+
+[Features]
+Stylus=true
+Reversible=true
+Touch=true
+Buttons=9
+Ring=true
+StatusLEDs=Ring
+
+[Buttons]
+Left=A;B;C;D;E;F;G;H;I
+
+Ring=I
+RingNumModes=4
diff --git a/data/intuos-pro-2-l.tablet b/data/intuos-pro-2-l.tablet
index 421b971..52cebcd 100644
--- a/data/intuos-pro-2-l.tablet
+++ b/data/intuos-pro-2-l.tablet
@@ -46,8 +46,8 @@
 Name=Wacom Intuos Pro 2 L
 DeviceMatch=usb:056a:0358
 Class=Intuos5
-Width=9
-Height=6
+Width=12
+Height=8
 Layout=intuos-pro-2-l.svg
 IntegratedIn=
 Styli=0x842;0x84a;0x160802;0x16080a;0x100802;0x10080a;0x140802;0x14080a;0x120802;0x100804;0x10080c;0x100902;0x10090a
diff --git a/data/intuos-pro-2-m-wl.tablet b/data/intuos-pro-2-m-wl.tablet
new file mode 100644
index 0000000..d7d6694
--- /dev/null
+++ b/data/intuos-pro-2-m-wl.tablet
@@ -0,0 +1,67 @@
+# Wacom
+# Intuos Pro 2 M WL
+# PTH-660
+#
+# Button Map:
+# (A=1, B=2, C=3, ...)
+#
+#    *-----------------------*
+#    |                       |
+#  A |                       |
+#  B |                       |
+#  C |                       |
+#  D |                       |
+#  I |        TABLET         |
+#  E |                       |
+#  F |                       |
+#  G |                       |
+#  H |                       |
+#    |                       |
+#    *-----------------------*
+#
+# Touch Ring Map:
+# (A=1st ring, B=2nd ring, ...)
+#
+#    *-----------------------*
+#    |                       |
+#  A |        TABLET         |
+#    |                       |
+#    *-----------------------*
+#
+# LED Map:
+# (XY=Bank X, LED Y; *=Invisible)
+#
+#        *-----------------------*
+#        |                       |
+#        |                       |
+#  01 02 |                       |
+#        |        TABLET         |
+#  00 03 |                       |
+#        |                       |
+#        |                       |
+#        *-----------------------*
+#
+
+[Device]
+Name=Wacom Intuos Pro 2 M WL
+DeviceMatch=bluetooth:056a:0360
+Class=Intuos5
+Width=9
+Height=6
+Layout=intuos-pro-2-m.svg
+IntegratedIn=
+Styli=0x842;0x84a;0x160802;0x16080a;0x100802;0x10080a;0x140802;0x14080a;0x120802;0x100804;0x10080c;0x100902;0x10090a
+
+[Features]
+Stylus=true
+Reversible=true
+Touch=true
+Buttons=9
+Ring=true
+StatusLEDs=Ring
+
+[Buttons]
+Left=A;B;C;D;E;F;G;H;I
+
+Ring=I
+RingNumModes=4
-- 
2.13.5

